website/content/docs/v0.7/Learn More/talosctl.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "talosctl"
+weight: 7
+---
+
+The `talosctl` tool packs a lot of power into a small package.  It acts as a
+reference implementation for the Talos API, but it also handles a lot of
+conveniences for the use of Talos and its clusters.
+
+## Client Configuration
+
+Talosctl configuration is located in `$XDG_CONFIG_HOME/talos/config.yaml` if
+`$XDG_CONFIG_HOME` is defined.  Otherwise it is in `$HOME/.talos/config`.  The
+location can always be overridden by the `TALOSCONFIG` environment variable or the
+`--talosconfig` parameter.
+
+Like `kubectl`, `talosctl` uses the concept of configuration contexts, so any
+number of Talos clusters can be managed with a single configuration file.
+Unlike `kubectl`, it also comes with some intelligent tooling (inspired by Joey
+Espinosa's excellent `kconf` tool) to manage the merging of new contexts into
+the config.  The default operation is a non-destructive merge, where if a
+context of the same name already exists in the file, the context to be added is
+renamed by appending an index number.  You can easily overwrite instead, as
+well.  See the `talosctl config help` for more information.
+
+## Endpoints and Nodes
+
+![Endpoints and Nodes](/images/endpoints-and-nodes.png)
+
+Endpoints are the communication endpoints to which the client directly talks.
+These can be load balancers, DNS hostnames, a list of IPs, etc.
+In general, it is recommended that these point to the set of control plane
+nodes, either directly or through a reverse proxy or load balancer.
+
+Each endpoint will automatically proxy requests destined to another node through
+it, so it is not necessary to change the endpoint configuration just because you
+wish to talk to a different node within the cluster.
+
+Endpoints _do_, however, need to be members of the same Talos cluster as the
+target node, because these proxied connections reply on certificate-based
+authentication.
+
+The Node is the target node on which you wish to perform the API call.
```

Comment:
Not an English major, but Node is both capitalized and not capitalized in this sentence.

Reply:
It sounded like a Good Idea at the time.

Reply:
Realistically, one is being used as a proper noun, while the other is a generic noun.  It is a way of visually differentiating the different imputations.  It is also a convention I generally use:  if something is an archetype, I capitalize it.  However, in this case, we can handle that simply by using backticks instead.

Reply:
I trust your English knowledge more than mine. I will go with whatever you think is best, including ignoring me 🙂.